packages/components/typography/stories/body.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import { Body as BodyComponent } from "../src";
+import { ThemeProvider } from "styled-components";
+import { theme } from "@buoysoftware/anchor-theme";
+
+export default {
+  title: "Typography / Components / Body",
+  component: BodyComponent,
+  argTypes: {
+    as: {
+      options: ["p", "span"],
+      control: { type: "select" },
+    },
+    size: {
+      options: ["l", "m", "s"],
+      control: { type: "select" },
```

Comment:
Are you able to add argument types for strong and emphasized?

Reply:
They are automatically provided.

Reply:
pshhh awesome